lib/cc/analyzer/engines_builder.rb

```diff
@@ -44,7 +44,7 @@ def engine_config(raw_engine_config)
 
       def names_and_raw_engine_configs
         {}.tap do |ret|
-          @config.engines.each do |name, raw_engine_config|
+          (@config.engines || []).each do |name, raw_engine_config|
```

Comment:
I guess we could. Is it worth it, though? It's longer to write, and `wrap` has some surprising semantics, and the returned object is a YAML node that already responds to `each`, so why wrap it if it's already present?